### ISSUE TITLE
Add `extends` option

### ISSUE THREAD

**Issue by SamVerschueren** [SamVerschueren/clinton#9]
Add possibility to extend the configuration with the `extends` option like [ESLint](http://eslint.org/docs/user-guide/configuring#extending-configuration-files).

**Comment by SamVerschueren** [comment]
Since #13 is implemented, this one should be quite easy to do.

**Comment by voxpelli** [comment]
Much 👍 on this

Just found this project after @sindresorhus starred it and would like to look into using it in all of our internal projects, but since they are so many, I would prefer to have a single shared internal config, like I currently do with `eslint-config-standard` – wrap it in an internal module that adds some extras and then have all internal projects rely on that.

Is this still something you are interested in implementing @SamVerschueren?

**Comment by SamVerschueren** [comment]
Yes, would be happy to have this feature. Just not sure if I can find the time to do it so help is always welcome.
